patches/tModLoader/Terraria/GameContent/Bestiary/BestiaryDatabase.cs.patch

```diff
@@ -1,12 +1,40 @@
 --- src/Terraria/Terraria/GameContent/Bestiary/BestiaryDatabase.cs
 +++ src/tModLoader/Terraria/GameContent/Bestiary/BestiaryDatabase.cs
-@@ -1,5 +_,6 @@
+@@ -1,16 +_,19 @@
  using System.Collections.Generic;
  using Terraria.GameContent.ItemDropRules;
 +using Terraria.ModLoader;
  
  namespace Terraria.GameContent.Bestiary
  {
+-	public class BestiaryDatabase
++	public partial class BestiaryDatabase
+ 	{
+ 		public delegate void BestiaryEntriesPass(BestiaryEntry entry);
+ 
+ 		private List<BestiaryEntry> _entries = new List<BestiaryEntry>();
++		private List<BestiaryEntry> _vanillaEntries = new List<BestiaryEntry>();
```

Comment:
Why not define _vanillaEntires and _byMod TML's partial class instead of directly in here?